go.mod

```diff
@@ -6,13 +6,15 @@ require (
 	github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46
 	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
 	github.com/emicklei/go-restful/v3 v3.8.0
+	github.com/getkin/kin-openapi v0.76.0
```

Comment:
I recall that we did a fair bit of work to ensure that kin-openapi isn't getting imported in k/k. Should the fuzzer testing be moved into integration instead?